### PR TITLE
refactor: Check references for assigned TimeKeeper references (#2)

### PR DESCRIPTION
**Pull request by TypAnna**
Changes from seconds to milliseconds in relevant places (think I found all 2nd/3rd hand references as well).

Kept deegre per second due to standard unit (Hz)

Fixes partly #2

### CODE REVIEW

File: src/assets/scripts/client/aircraft/AircraftConflict.js [hunk 1]
```diff
@@ -126,8 +127,8 @@ export default class AircraftConflict {
 
         // TODO: replace magic numbers with enum
         // Ignore aircraft in the first minute of their flight
-        if (gameTime - this.aircraft[0].takeoffTime < 60 ||
-            gameTime - this.aircraft[1].takeoffTime < 60) {
+        if (gameTime - this.aircraft[0].takeoffTime < ONE_MINUTE_IN_MILLISECONDS ||
```
Comment (willwull) [inline src/assets/scripts/client/aircraft/AircraftConflict.js]: Shouldn't this be `TIME.ONE_MINUTE_IN_MILLISECONDS`?

File: src/assets/scripts/client/aircraft/AircraftConflict.js [hunk 2]
```diff
@@ -126,8 +127,8 @@ export default class AircraftConflict {
...
+            gameTime - this.aircraft[1].takeoffTime < ONE_MINUTE_IN_MILLISECONDS) {
```
Comment (willwull) [inline src/assets/scripts/client/aircraft/AircraftConflict.js]: Same here

File: src/assets/scripts/client/aircraft/AircraftModel.js [hunk 3]
```diff
@@ -2239,7 +2240,7 @@ export default class AircraftModel {
                 offsetGameTime
             ]);
             // TODO: this can be abstracted
-        } else if (abs(offsetGameTime - this.relativePositionHistory[this.relativePositionHistory.length - 1][2]) > 4 / GameController.game_speedup()) {
+        } else if (abs(offsetGameTime - this.relativePositionHistory[this.relativePositionHistory.length - 1][2]) > 4000 / GameController.game_speedup()) {
```
Comment (willwull) [inline src/assets/scripts/client/aircraft/AircraftModel.js:2243]: Do we know what `4` means in this context? I can't really understand the original code lol.

Comment (TypAnna) [reply src/assets/scripts/client/aircraft/AircraftModel.js:2243]: Yea, I thought the same, but I investigated and it was time in seconds, and is now time in milliseconds:)

**Review by willwull — APPROVED** [review]
💯👌👀

**Pull request merged**